compiler/luci-eval-driver/src/EvalDriver.cpp

```diff
@@ -90,8 +90,20 @@ int entry(int argc, char **argv)
   const char *input_prefix = argv[3];
   const char *output_file = argv[4];
 
+  // define model data
+  std::ifstream fs(filename, std::ifstream::binary);
+  if (fs.fail())
+  {
+    throw std::runtime_error("Cannot open model file \"" + std::string(filename) + "\".\n");
+  }
+
+  // model buffer (must be alive longer than importer and interpreter)
+  std::vector<char> model_data((std::istreambuf_iterator<char>(fs)),
+                               std::istreambuf_iterator<char>());
```

Comment:
(1) This seems very important condition. How do we let users of luci-import provide this?
(2) `std::vector<char` looks like make a copy content from the file and this seems misleading of not uising constant copy. actully I am confused for this.

Reply:
> (1) This seems very important condition. How do we let users of luci-import provide this? (2) `std::vector<char` looks like make a copy content from the file and this seems misleading of not uising constant copy. actully I am confused for this.

1. I agree with you and this condition confuses me too. I wrote about this in https://github.com/Samsung/ONE/pull/8043#discussion_r754029148 and in https://github.com/Samsung/ONE/issues/8042#issuecomment-974442010. I think we need to enable constant copying by default (as done in draft) and notify user via comments in Importer.h about this. Or introduce another approach or close this draft.
2. This behavior is not confuses me because usage scenario of interpreter on embedded system differs from usage on more powerfull devices. Usually, embedded systems works without file system, so, loading of model to array may be done in speciall .cpp file (e.g. https://github.com/tensorflow/tflite-micro/blob/fdc88436a1928b2314caf5e1445fdf16ccd83d68/tensorflow/lite/micro/examples/hello_world/model.cc) before compilation.

Reply:
>  I think we need to enable constant copying by default

This is the current behaviour. I would like not to name this to `copy` and name some for the new one.
You've used `Remote` to express the difference of existing CircleConst. Something like, but not `Remote`.

> Or introduce another approach or close this draft.

For safety, I prefer another approach if possible, but have to see...

> This behavior is not confuses me

The import thing is I am confused.

> Usually, embedded systems works without file system

OK, thanks for this sample code. But I cannot image using `vector` can be connected to the const byte buffer. You have not provided any clue.